lib/grapes/com.google.code.google-collections/google-collect/ivy-snapshot-20080530.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
how does the PluginWizard use google-collect library?

Reply:
I have added a sample procedure with groovy dependencies.